decidim-core/config/locales/en.yml

```
@@ -588,6 +588,9 @@ en:
           sign_up: Please complete your profile
           subtitle: Please fill in the following form in order to complete the sign up
           username_help: Public name that appears on your posts. With the aim of guaranteeing the anonymity, can be any name.
+        new_tos_fields:
+          sign_up: Sign up
+          sign_up_title: Finish sign up
```

Please align terminology to develop, as a result of #13076
```suggestion
        new_tos_fields:
          sign_up: Create an account
          sign_up_title: Finish creating your account
```